Server/Plugins/APIDump/Classes/BlockEntities.lua

```diff
@@ -1401,13 +1449,79 @@ World:ForEachChestInChunk(Player:GetChunkX(), Player:GetChunkZ(),
 				},
 				Notes = "Sets the spawn delay.",
 			},
+			SetSpawnCount =
+			{
+				Params =
+				{
+					{
+						Name = "SpawnCount",
+						Type = "number",
+					},
+				},
+				Notes = "Sets the number of entities the spawner will try to spawn in each activation. Might not spawn all of them due to spawn limitations of the entity.",
+			},
+			SetSpawnRange =
+			{
+				Params =
+				{
+					{
+						Name = "SpawnRange",
+						Type = "number",
+					},
+				},
+				Notes = "Sets the diameter of the square the spawner will try to spawn entities in.",
```

Comment:
Radius, not diameter. You should also say it'll be clipped to 20.

Reply:
This hasn't been resolved?